modules/custom/activity_viewer/src/Plugin/views/filter/ActivityPostVisibilityAccess.php

```diff
@@ -79,15 +79,21 @@ public function query():void {
     $account = $this->view->getUser();
 
     $open_groups = [];
+    $public_groups = [];
     $group_memberships = [];
 
     if ($this->moduleHandler->moduleExists('social_group')) {
       // @todo This creates a dependency on Social Group which shouldn't exist,
       // this access logic should be in that module instead.
       $open_groups = social_group_get_all_open_groups();
+      $public_groups = social_group_get_all_public_groups();
       $group_memberships = $this->groupHelper->getAllGroupsForUser($account->id());
     }
-    $groups = array_merge($open_groups, $group_memberships);
+    $groups = [
+      ...$open_groups,
```

Comment:
This interesting way of merging an array. Do you have some documentation for this? I would be interested to learn.

Reply:
Sure, you can check these:
- https://www.kindacode.com/article/merging-arrays-in-php/
- https://www.phptutorial.net/php-tutorial/php-spread-operator/

Reply:
Thank you for documentation.